pkg/cmd/extension/manager.go

```diff
@@ -249,6 +250,32 @@ func (m *Manager) installDir() string {
 	return filepath.Join(m.dataDir(), "extensions")
 }
 
+func (m *Manager) Create(name string) error {
+	exe, err := m.lookPath("git")
+	if err != nil {
+		return err
+	}
+
+	err = os.Mkdir(name, 0755)
+	if err != nil {
+		return err
+	}
+
+	initCmd := m.newCommand(exe, "init", "-b", "main", "--quiet", name)
+	err = initCmd.Run()
+	if err != nil {
+		return err
+	}
+
+	fileTmpl := heredoc.Docf(`
+		#!/bin/bash
+		echo "Hello %[1]s"
+	`, name)
```

Comment:
Do we want to add anything else to the template?

Reply:
yes! I think we should have:

- example of checking to see if something is on path (like `ruby`, `python`, or `go`)
- example of passing args through to something (to demonstrate use of `"$@"`)
- example of using `gh api`

Reply:
@vilmibm I added some examples. Let me know what you think.